config.defaults.json

```diff
@@ -23,6 +23,11 @@
             "enabled": false,
             "botToken": "See README",
             "chats": {}
+        },
+        "ifttt": {
+            "enabled": false,
+            "webhookKey": "see README",
+            "webhookEvent": "see README"
```

Comment:
This doesn't need a setting, but can just be hardcoded to `too_good_to_go_updated`.